test_conformance/compiler/test_feature_macro.cpp

```diff
@@ -833,3 +834,58 @@ int test_features_macro(cl_device_id deviceID, cl_context context,
 
     return error;
 }
+
+// This test checks that a supported feature comes with other required features
+int test_features_macro_coupling(cl_device_id deviceID, cl_context context,
+                                 cl_command_queue queue, int num_elements)
+{
+    log_info("  testing for required features coupling\n");
```

Comment:
I would suggest dropping this `log_info` call; the test harness will already print
```
features_macro_coupling...                                                                               
features_macro_coupling passed
```
which should be informative enough.